iter/iter.go

```diff
@@ -7,12 +7,10 @@ import (
 // Iter is a simple wrapper around a tsz.Iter to make debug logging easier
```

Comment:
this comment is no longer true. the purpose is now simply to abstract away go-tsz